src/mockData/mockData.ts

```diff
@@ -1,8 +1,17 @@
 // locale will be used for the site's language settings
 
+import {
+  Degree,
+  Facility,
+  Insurance,
+  Language,
+  HealthcareProfessional,
+  Specialty,
+} from '../typeDefs/gqlTypes';
+
 // language is the spoken language
 
-const facilities = [
+const facilities: Array<Facility> = [
```

Comment:
nice! likes like the types are all in order.

Is the updated typescript types file being updated and checked into git  as we change the schema? As it affects the types used across the server, we'll need to make sure it's in sync